docs/topics/Gateway.md

```diff
@@ -772,14 +772,25 @@ A user's presence is their current state on a guild. This event is sent when a u
 
 ###### Presence Update Event Fields
 
-| Field      | Type                                                              | Description                                  |
-| ---------- | ----------------------------------------------------------------- | -------------------------------------------- |
-| user       | [user](#DOCS_RESOURCES_USER/user-object) object                   | the user presence is being updated for       |
-| roles      | array of snowflakes                                               | roles this user is in                        |
-| game       | ?[activity](#DOCS_TOPICS_GATEWAY/activity-object) object          | null, or the user's current activity         |
-| guild_id   | snowflake                                                         | id of the guild                              |
-| status     | string                                                            | either "idle", "dnd", "online", or "offline" |
-| activities | array of [activity](#DOCS_TOPICS_GATEWAY/activity-object) objects | user's current activities                    |
+| Field         | Type                                                               | Description                                  |
+| ------------- | ------------------------------------------------------------------ | -------------------------------------------- |
+| user          | [user](#DOCS_RESOURCES_USER/user-object) object                    | the user presence is being updated for       |
+| roles         | array of snowflakes                                                | roles this user is in                        |
+| game          | ?[activity](#DOCS_TOPICS_GATEWAY/activity-object) object           | null, or the user's current activity         |
+| guild_id      | snowflake                                                          | id of the guild                              |
+| status        | string                                                             | either "idle", "dnd", "online", or "offline" |
+| activities    | array of [activity](#DOCS_TOPICS_GATEWAY/activity-object) objects  | user's current activities                    |
+| client_status | ?[client_status](#DOCS_TOPICS_GATEWAY/client-status-object) object | user's platform-dependent status             |
+
+#### Client Status Object
+
+All status strings are those that are documented in the original presence update status field; that is, "idle", "dnd", "online", or "offline".
```

Comment:
Offline is not an option. If offline on a given platform it will just not be present.

Reply:
Fixed.